src/platform/packages/shared/kbn-esql-utils/src/utils/query_parsing_helpers.ts

```diff
@@ -138,9 +139,9 @@ export const isQueryWrappedByPipes = (query: string): boolean => {
   return numberOfCommands === pipesWithNewLine?.length;
 };
 
-export const prettifyQuery = (query: string, isWrapped: boolean): string => {
-  const { root } = parse(query);
-  return BasicPrettyPrinter.print(root, { multiline: !isWrapped });
+export const prettifyQuery = (src: string): string => {
+  const { root } = Parser.parse(src, { withFormatting: true });
+  return WrappingPrettyPrinter.print(root, { multiline: true });
```

Comment:
`multiline` is not necessary here, it will automatically be multiline, if the query is too long. If it first, though, in one line, it can leave multiple command on one line.

```suggestion
  return WrappingPrettyPrinter.print(root);
```

Reply:
So if I have `FROM index | limit 10 ` it will leave them at the same line? I think I prefer to wrap them every time, no?